habitat/tasks/nav/nav.py

```diff
@@ -430,7 +430,7 @@ def reset_metric(self, *args: Any, episode, task, **kwargs: Any):
         task.measurements.check_measure_dependencies(
             self.uuid, [DistanceToGoal.cls_uuid]
         )
-        self.update_metric(*args, episode=episode, task=task, **kwargs)
+        self.update_metric(episode=episode, task=task, *args, **kwargs)
```

Comment:
THis modification makes the positioning of `args, kwargs` in function signature consistent with the common python conventions.